### PR TITLE
Adding case insensitive inclusion validator

### PR DESCRIPTION
**Pull request by moizjv**

### CODE REVIEW

File: lib/desired-caps.js [hunk 1]
```diff
@@ -110,6 +110,17 @@ validator.validators.deprecated = function (value, options, key) {
   }
   return null;
 };
+validator.validators.inclusionCaseInsensitive = function (value, options) {
+  if (typeof value !== 'string') {
+    return `unrecognised`;
```
Comment (imurchie) [inline lib/desired-caps.js]: This can be a plain old string. Also, what output comes when this happens?

Comment (moizjv) [inline lib/desired-caps.js]: for consistency with above used strings i added that quotes. Output will be like ` platformName unrecognised.`

Comment (imurchie) [inline lib/desired-caps.js]: Ah. That is a good output. 

As for the strings, the rest should be changed rather than another be added in the wrong format.

**Comment by imurchie** [comment]
Looks good!

Any ideas about the failing download of node?

**Comment by moizjv** [comment]
@imurchie ok now it passed; though coverage is complaining, is it good to merge?

**Comment by imurchie** [comment]
Haha! Coverage went up and coveralls is complaining.

Yes, merge away.

**Pull request merged**